### PR TITLE
revert to original ngram normalization in FastTextKeyedVectors#adjust…

### LINKED ISSUE #2537: FastText inconsistent normalization in adjust_vectors method vs word_vec
#### Problem description

There is inconsistency in normalization in ```FastTextKeyedVectors#adjust_vectors``` vs ```WordEmbeddingsKeyedVectors#word_vec```

#### Steps/code/corpus to reproduce

adjust_vectors:
```
            word_vec /= len(ngram_hashes) + 1
```

word_vec method:
```
            return word_vec / len(ngram_hashes)
```

#### Versions

```
Linux-4.18.0-24-generic-x86_64-with-Ubuntu-18.04-bionic
Python 3.7.1 (default, Oct 22 2018, 11:21:55) 
[GCC 8.2.0]
NumPy 1.16.3
SciPy 1.2.1
gensim 3.7.3
FAST_VERSION 1
```

### PR DESCRIPTION
**Pull request by vackosar**
Implemented as suggested by https://github.com/RaRe-Technologies/gensim/issues/2537#issuecomment-506584926
Fixes #2537

### CODE REVIEW

**Comment by mpenkov** [comment]
The tests appear to be failing. Could you please have a look?

I suspect the reason for the failures is either:

1. Test data is incorrect. We need to update the tests.
2. Test data is correct, and we need to roll back the changes in the implementation.

**Comment by mpenkov** [comment]
Ping @vackosar

File: gensim/models/keyedvectors.py [hunk 1]
```diff
@@ -2291,7 +2291,7 @@ def adjust_vectors(self):
             ngram_hashes = ft_ngram_hashes(w, self.min_n, self.max_n, self.bucket, self.compatible_hash)
             for nh in ngram_hashes:
                 word_vec += self.vectors_ngrams[nh]
-            word_vec /= len(ngram_hashes) + 1
```
Comment (gojomo) [inline gensim/models/keyedvectors.py:2294]: This "fix" is plainly incorrect: the value of `word_vec` is composed from 1 full-word vector (from `vectors_vocab`), plus `len(ngram_hashes)` ngram vectors. So to get the average of `len(ngram_hashes) + 1` separate vectors, you divide by `len(ngram_hashes) + 1`.

**Comment by mpenkov** [comment]
Closing (see discussion in #2537 )

**Pull request closed without merging**